main.go

```diff
@@ -37,6 +40,9 @@ func main() { //nolint
 	go oracle.Start()
 	defer oracle.Stop()
 
+	monitorConfig := monitor.DefaultMonitorConfig
+	ms := monitor.New(&monitorConfig, filepath.Base(config.DefaultPluginConfFile))
```

Comment:
@raj-shekhar1 just a QQ: in this PR we introduced a perf report that can be dumped into a directory,  in our design we assumes to dump it into the directory where the oracle-sever's configuration file exist. In some special deployment context, for example in the k8s, we just grant the configuration file read access to oracle-server, is it allow to dump a file to the base directory of the configuration file without mounting it or granting the access right? If we cannot, I suppose that we should introduce a new conf field to nominate the directory which is prepared in advanced before dumping report in it.

Reply:
@Jason-Zhangxin-Chen , It will work but the file will not persists in the case of K8s between pod restarts.

> If we cannot, I suppose that we should introduce a new conf field to nominate the directory which is prepared in advanced before dumping report in it.

This looks better to me, as an appropriate directory can be pre-mounted with necessary permissions.

Reply:
Thank you so much, I will create a new PR to add this config.

Reply:
added a new config for profile dir.